src/app/builder/api/ansyn-api.service.ts

```diff
@@ -88,6 +92,9 @@ export class AnsynApi {
 		this.store.dispatch(new SelectCaseAction(this.casesConfig.defaultCase));
 	}
 
+	disableInitialSearch() {
```

Comment:
rename to enableInitialSearch(isEnabled: string)
make it a private function
call with constructor arg